extensions.json

```diff
@@ -1,7 +1,17 @@
 {
-  "tags": [
-    "Jump Start"
+  "categories": [
+    {
+      "id": "extension",
+      "title": "extensions",
+      "description": "Useful tools that utilize the Posit Connect API to extend the functionality of Connect."
+    },
+    {
+      "id": "example",
+      "title": "examples",
```

Comment:
I think that `id` and `title` is a confusing way to name these two peer values. There's nothing in that innately that tells me how they should be used, and nothing _structurally_ that tells me which I should use in my content. I think we should remove `id` for now in the spirit of YAGNI.

In general, I'm in favor of pluralizing the names and titles of categories and tags. Even displaying the category on the page of a content item, it would look fine and correct to have a label that read `category: Examples`.